docs/important/troubleshooting_en.md

```diff
@@ -0,0 +1,164 @@
+# Troubleshooting
+
+The Troubleshooting chapter provides assistance in locating possible errors during Dogu development.
+Solutions are also presented for known errors.
+
+## How do I avoid errors due to invalid configuration entries?
+
+- Valid registry paths are to be considered
+- `doguctl` can validate configuration values. See [Validation](relevant_functionalities_en.md#validation-and-default-values)
+- Overview of all keys: `etcdctl ls -r config/<doguname>`
+- From the container it is possible to check encrypted values:
+    - `docker exec -it redmine doguctl config -e sa-postgresql/username`
+
+## The dogu does not start and/or is in a restart loop.
+
+If there is an approximate guess about the position of the faulty location, additional `echo` output can help to analyze the problem.
+
+If the source of the error is still unclear, it is necessary to execute the statements inside the container one by one:
+- Shell into the container: `docker exec -it <doguname> bash`.
+- After that, the container can be debugged:
+    - Checking the filesystem
+    - Iterative execution of the commands in the scripts to locate the defect
+    - Repairing the defect using on-board means in the container
+
+During a restart loop, you have the following options to bring the container to a stable state:
+- During development, the best way you can do is to overwrite the entrypoint of the container.
+    - `ENTRYPOINT ["tail", "-f", "/dev/null"]` instead of `CMD ["/resources/startup.sh"]`.
+    - Alternatively, a `sleep --infinite` can be written anywhere in the startup script.
+- In production, a container must be stopped manually in a restart loop and restarted with a commit:
+```bash
+docker stop ldap
+# commit the stopped container to a local debug image
+docker commit hashFromYourStoppedContainer_cebf8699cd9d debug/ldap
+# the volume arguments can be generated with docker inspect
+docker inspect ldap -f '{{range.Mounts}}-v {{.Source}}:{{.Destination}} {{end}}'
+# debug your dogu container
+docker run -ti \
+--rm \
+# here goes the docker volume list from above
+-v /etc/ces:/etc/ces \
+-v /var/lib/ces/ldap/volumes/_private:/private \
+-v /var/lib/ces/ldap/volumes/config:/etc/openldap/slapd.d \
+-v /var/lib/ces/ldap/volumes/db:/var/lib/openldap \
+--entrypoint=/bin/sh --network cesnet1 --name ldap --hostname ldap \
+debug/ldap
+```
+
+## Logging output is missing or has wrong granularity
+
+There are many reasons why completely/partially missing logging output or the wrong logging output occurs.
```

Comment:
The mistake in the german version got directly translated to the english version.
```suggestion
There can be many reasons why there is completely/partially missing logging output or the wrong logging output occurs.
```